tests/unit-tests/importer/tests-donations.php

```diff
@@ -0,0 +1,407 @@
+<?php
+/**
+ * Meta
+ * @package Give\Tests\Importer
+ */
+
+
+class WC_Tests_Give_Import_Donations extends Give_Unit_Test_Case {
```

Comment:
What is this prefix? If you're forking then update the prefix...

Reply:
Updated the Class name to Tests_Give_Import_Donations